lib/v2/screens/dashboard/wallet_screen.dart

```diff
@@ -0,0 +1,112 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:seeds/v2/blocs/rates/viewmodels/bloc.dart';
+import 'package:seeds/v2/blocs/rates/viewmodels/rates_bloc.dart';
+import 'package:seeds/v2/components/profile_avatar.dart';
+import 'package:seeds/v2/constants/app_colors.dart';
+import 'package:seeds/v2/i18n/explore_screens/explore/explore.i18n.dart';
+import 'package:seeds/v2/navigation/navigation_service.dart';
+import 'package:seeds/v2/screens/dashboard/interactor/viewmodels/wallet_bloc.dart';
+import 'package:seeds/v2/screens/dashboard/interactor/viewmodels/wallet_event.dart';
+import 'package:seeds/v2/screens/dashboard/wallet_header.dart';
+import 'package:seeds/v2/design/app_theme.dart';
+
+// Widgets to be moved to v2
+import 'package:seeds/widgets/v2_widgets/dashboard_widgets/receive_button.dart';
+import 'package:seeds/widgets/v2_widgets/dashboard_widgets/send_button.dart';
+
+enum TransactionType { income, outcome }
+
+/// Wallet SCREEN
+class WalletScreen extends StatelessWidget {
+  const WalletScreen({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocProvider(
+      create: (_) => WalletBloc()..add(const RefreshDataEvent()),
+      child: RefreshIndicator(
+          child: Scaffold(
+            appBar: buildAppBar(context) as PreferredSizeWidget?,
+            body: ListView(
+              // TODO(n13): Use exact measurements from figma
+              children: <Widget>[
+                const SizedBox(height: 15),
+                const WalletHeader(),
+                const SizedBox(height: 20),
+                buildSendReceiveButton(context),
+                const SizedBox(height: 20),
+                walletBottom(context),
+              ],
+            ),
+          ),
+          onRefresh: () async {
+            BlocProvider.of<RatesBloc>(context)..add(const FetchRates());
+            BlocProvider.of<WalletBloc>(context)..add(const RefreshDataEvent());
+          }),
+    );
+  }
+
+  Widget buildSendReceiveButton(BuildContext context) {
+    return Container(
+      padding: const EdgeInsets.only(left: 20, right: 20),
+      child: Row(mainAxisSize: MainAxisSize.max, mainAxisAlignment: MainAxisAlignment.center, children: <Widget>[
+        Expanded(child: SendButton(onPress: () => NavigationService.of(context).navigateTo(Routes.transfer))),
+        const SizedBox(width: 20),
+        Expanded(child: ReceiveButton(onPress: () async => await NavigationService.of(context).navigateTo(Routes.receiveEnterDataScreen))),
+      ]),
+    );
+  }
+
+  // TODO(n13): Use exact measurements for sizes of app bar elements from figma, these are guesses
+  Widget buildAppBar(BuildContext context) {
+    return AppBar(
+      titleSpacing: 0,
+      leading: Container(
```

Comment:
@gguij004  is to work on this UI i guess?

Reply:
ui is mostly done, but yes I will help and test any missing pieces.

Reply:
Great if you can do it @gguij004